traits/trait_set_object.py

```diff
@@ -15,6 +15,7 @@
 
 from traits.trait_base import _validate_everything
 from traits.trait_errors import TraitError
+from traits.observers._i_observable import IObservable as _IObservable
```

Comment:
Import order... and I want to get rid this preceding underscore. Will do that in the separate PR that does the same thing to all the other observables.

Reply:
https://github.com/enthought/traits/pull/1085